README.md

```diff
@@ -132,6 +136,9 @@ if ([crashReporter hasPendingCrashReport]) {
 import CrashReporter
 
 ...
+#if DEBUG
```

Comment:
`DEBUG` is not defined by default in Swift, so this check (as example) won't work. See https://stackoverflow.com/a/24152730/1703290

Reply:
Also, `DEBUG` mode doesn't always mean that debugger is attached.
Let's use something like this:
```
// Uncomment and implement isDebuggerAttached to safely run this code with a debugger.
// See: https://github.com/microsoft/plcrashreporter/blob/2dd862ce049e6f43feb355308dfc710f3af54c4d/Source/Crash%20Demo/main.m#L96
// if (!isDebuggerAttached()) {
...
// }
```